### PR TITLE
in progress...Needs help

### PR DESCRIPTION
**Pull request by chishko**
This is a work in progress i had it running for a little and wrote a partial config file. Then i got errors. Thought i should submit something

### CODE REVIEW

**Comment by chishko** [comment]
i still need to add arguments too

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+int main(int argc, char *argv[]){
+// init
+    std::ofstream outputFile;
+    outputFile.open (CONFIG_FILE_WRITE_PATH);
+
+    string lastName;
+    cout << "Enter last name:" << endl;
+    getline(cin,lastName);
+    if(lastName == "")
```
Comment (ninjapanzer) [inline main.cpp]: Around each of these inputs you should consider doing a loop so it continues to reprompt the user if they continue to not enter anything

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+using std::getline;
+
+const string CONFIG_FILE_WRITE_PATH = "/Users/Jon/Desktop/C++repos/CPT-180-27-Assignment-2-Config-Generator/config.txt";
+
+int main(int argc, char *argv[]){
+// init
```
Comment (ninjapanzer) [inline main.cpp]: Right so you need a condition here in relationship to `argv` that will tell us if the command line argument was `init` or `edit`

File: main.cpp [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+    outputFile.close();
+
+
+    //edit
+
+    outputFile.open (CONFIG_FILE_WRITE_PATH);
+
+
+    cout << "Edit last name:" << endl;
```
Comment (ninjapanzer) [inline main.cpp:96]: these need a condition in regards to `argv[2]` that states what the field the user would like to update is

File: main.cpp [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    if (knownRecipientsFile == "") {
+        new_outputfile<< "[knowrecipients] = knownrecipient.txt" << endl;
+    } else
+        new_outputfile << "[knowrecipients] = " << knownRecipientsFile << endl;
+}
+main(int argc, char *argv[]);
+    if(static_cast<string>(argv[1]) == "init"){
+        init();
+}
+    else(static_cast<string>(argv[2]) == "edit"){
```
Comment (ninjapanzer) [inline main.cpp:151]: also `argv[1]`

**Pull request closed without merging**